restapi/authors.go

```diff
@@ -36,7 +36,7 @@ func (api api) CreateAuthors(w http.ResponseWriter, r *http.Request) render.Rend
 	return rest.MsgStatusOK("Authors added successfully")
 }
 
-func (api api) ReturnAllCountries(w http.ResponseWriter, r *http.Request) render.Renderer {
+func (api api) ReturnAllUsedCountries(w http.ResponseWriter, r *http.Request) render.Renderer {
```

Comment:
This name is kind of misleading, lets use `ReturnAuthorOriginCountries` or something along those lines